ext.gradle

```diff
@@ -29,10 +29,10 @@ final def SPINE_VERSION = '0.10.43-SNAPSHOT'
 
 ext {
     spineVersion = SPINE_VERSION
+    versionToPublish = SPINE_VERSION
 
     protobufVersion = '3.5.1'
```

Comment:
Please try to use the [dependencies.gradle](https://github.com/SpineEventEngine/config/blob/409b966f2784f12b6a94fda95570018b879e5524/gradle/dependencies.gradle) instead.

There is a related issue, see #124. The problem is that the dependencies in this file are referenced from a bunch of integration-test projects. That's why this version still remains.

Please see if resolving #124 in this PR is easy for you. If not, let's merge this PR and open a new one.